README.md

```diff
@@ -17,6 +17,8 @@ Darcula theme for [Visual Studio Code](https://code.visualstudio.com) based on [
 * Dockerfile
 * Markdown
 * Properties
+* Rust
```

Comment:
Since this is a programming language, please move it after Groovy. This way we keep programming languages first in the list.